controllers/provider-aws/pkg/controller/controlplane/valuesprovider.go

```diff
@@ -292,3 +293,10 @@ func getCCMChartValues(
 
 	return values, nil
 }
+
+// getVerticalPodAutoscalerObject returns Unstructured object describing VerticalPodAutoscaler of group "autoscaling.k8s.io"
+func getVerticalPodAutoscalerObject() *unstructured.Unstructured {
+	obj := &unstructured.Unstructured{}
+	obj.SetGroupVersionKind(schema.GroupVersionKind{Group: "autoscaling.k8s.io", Kind: "VerticalPodAutoscaler", Version: "v1beta2"})
```

Comment:
Can you try to reuse `SchemeGroupVersion` from https://github.com/kubernetes/autoscaler/blob/5ff9f7b8ba65cfec027da647aff79f63c1176fbe/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1beta2/register.go#L26?

Reply:
I tried, but I cannot make gardener-extensions to depend on this project, I cannot build the correct project:version